gradio/components/annotated_image.py

```diff
@@ -141,12 +144,12 @@ def postprocess(
             base_img = np.array(PIL.Image.open(base_img))
         elif isinstance(base_img, np.ndarray):
             base_file = processing_utils.save_img_array_to_cache(
-                base_img, cache_dir=self.GRADIO_CACHE
+                base_img, cache_dir=self.GRADIO_CACHE, format=self.format or "png"
```

Comment:
If we make the change where format is, by default, `png`, we won't have to handle the case where it has to be `None` here and below

Reply:
Yea good point